openjdk-19.yaml

```diff
@@ -0,0 +1,124 @@
+#nolint:bad-version
+package:
+  name: openjdk-19
+  version: 19.0.2.7
+  epoch: 1
+  description:
+  copyright:
+    - license: GPL-2.0-only
+  dependencies:
+    runtime:
+      - freetype
+      - fontconfig
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - file
+      - freetype-dev
+      - cups-dev
+      - libx11-dev
+      - libxext-dev
+      - libxrender-dev
+      - libxrandr-dev
+      - libxtst-dev
+      - libxt-dev
+      - alsa-lib-dev
+      - libffi-dev
+      - bash
+      - zip
+      - fontconfig-dev
+      - libxi-dev
+      - libjpeg-dev
+      - giflib-dev
+      - lcms2-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: \.(\d+)$
+    replace: +$1
+    to: mangled-package-version
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/openjdk/jdk19u/archive/refs/tags/jdk-${{vars.mangled-package-version}}.tar.gz
+      expected-sha256: 5903efd527dd08e9c235c8822e3d5699c3d18a8618c3e533307e8d6491ffbbf0
+
+# We need pre-built binaries to use as Boot JDK https://hg.openjdk.org/jdk/jdk12/raw-file/tip/doc/building.html#boot-jdk-requirements
+  - uses: fetch
+    with:
+      uri: https://github.com/adoptium/temurin19-binaries/releases/download/jdk-19.0.2%2B7/OpenJDK19U-jdk_x64_linux_hotspot_19.0.2_7.tar.gz
```

Comment:
I wonder if we can use the existing `openjdk-17` package for the bootstrap rather than downloading the temurin binaries?  The idea is we want to build everything in wolfi from source.  There has been a need at times to use a binary to bootstrap a build but I'm wondering if we don't need that here and adding `openjdk-17` to the `environment.contents.packages` list would be enough?

Reply:
We can use openjdk-17 to build openjdk-18, then use 18 to build 19, and so on, because you can't use 17 to build 19 or 20

Reply:
What we can do is : 
- I can add openjdk-18 the same way
- I can prepare another PR for when we will have proper package, to use them instead of temurin one.

What i'm wondering is if we really need to rebuild the Java binaries instead of just adding it to the image, is there some value added on this ?

Reply:
It's important to us that we have the capability to patch and customize if need be, and that the build environment for our packages is Wolfi-ized (up-to-date and minimal). That's not to say that we are against potentially providing binary packages of certain JDK distributions. For now though we're going to continue building one ourselves.

@rawlingsj WDYT about bootstrapping using the Temurin JDK n-1 binary for now, and once we get to packaging N-1 we can swap that out? 

@vr could you swap the jlink based JRE to using the submodule approach [here](https://github.com/wolfi-dev/os/pull/906/files)?